src/MentorBot.Functions/App/Startup.cs

```diff
@@ -17,12 +20,16 @@ namespace MentorBot.Functions.App
     public static class Startup
     {
         /// <summary>Registers the services.</summary>
-        internal static void RegisterServices(IServiceCollection services)
+        internal static void RegisterServices(IServiceCollection services, IConfiguration configuration)
         {
             services.AddTransient<IHangoutsChatService, HangoutsChatService>();
             services.AddTransient<ICognitiveService, CognitiveService>();
             services.AddTransient<ICommandProcessor, LocalTimeProcessor>();
             services.AddTransient<IStringLocalizer, StringLocalizer>();
+
+            services.AddTransient<DbContext, ApplicationDbContext>();
+            services.AddDbContextPool<ApplicationDbContext>(optionsAction => optionsAction.UseSqlServer(configuration.GetConnectionString("DefaultConnection")));
```

Comment:
How to move **DefaultConnection** to localization or something?

Reply:
Why would you wanna do that?